src/components/ebay-carousel/utils/scroll-end/index.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Calls a function every time scrolling completes for an element.
+ * Returns a function to stop listening for scroll ends events.
+ *
+ * (Only works in mobile and relies on touch events)
+ *
+ * @param {HTMLElement} el The element which scrolls.
+ * @param {(offset: number)=>{}} fn The function to call after scrolling completes.
+ * @return {function} A function to cancel the scroll listener.
+ */
+module.exports = function onScrollEnd(el, fn) {
+    let frame;
+    let stage = 0;
+    el.addEventListener('touchmove', handleTouchMove);
```

Comment:
Should we make these [passive](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Improving_scrolling_performance_with_passive_listeners) events listeners, to ensure more smoothness?

Reply:
Should not be needed in this case as the event handler is removed immediately after the first touch move [here](https://github.com/eBay/ebayui-core/blob/6fcd95b6ea92c540f19e0eef8b0b9863bbf22f19/src/components/ebay-carousel/utils/scroll-end/index.js#L21). Then It waits for a touch end.